### DISCUSSION TITLE
Where can I found tests for node_worker.cc

### DISCUSSION THREAD

**Discussion by jdiaz-dev** [nodejs/help#5030] [Q&A]
I would like to contribute to c++ code, first I want to contribute to tests for node_worker.cc file in Node.js core but I can't find file where test are.

**Comment by bnoordhuis** [comment]
There are _many_, `git grep worker_threads test/` will find them for you. I count 200+ files, discounting hits in test/fixtures.

**Reply by jdiaz-dev** [reply]
The most files are for javascript. I would like to focus in test unit for worker in c++

**Reply by bnoordhuis** [reply]
There aren't any, and they arguably wouldn't add much value either. Node's test suite is heavily slanted towards integration testing.
